lib/router/case.js

```diff
@@ -1,17 +1,16 @@
 const { Router } = require('express');
 
-const { NotFoundError, InvalidRequestError } = require('../errors');
-const Case = require('../db/case');
+const Case = require('../models/case');
 
 module.exports = ({ hooks }) => {
 
   const router = Router();
 
   router.param('case', (req, res, next, id) => {
-    Case.query().findById(id)
+    Case.find(id)
       .then(model => {
         req.case = model;
```

Comment:
Small one - it looks like `Case.find()` returns an instance of `Case` rather than a model so I'd probably rename the var here.

Reply:
I'm going to ignore this because you cant have `case` as a variable name because it's a reserved word.